packages/devtools/lib/src/timeline/event_details.dart

```diff
@@ -30,6 +32,12 @@ bool _defaultCollapseNativeSamplesValue = false;
 Stream<bool> get onCollapseNativeSamplesEvent =>
     _collapseNativeSamplesController.stream;
 
+final _loadCpuProfileSnapshotController =
```

Comment:
this name is a bit long. Can we streamline it and also add a type to the LHS.
For example
`_profileSnapshots`
or
`_profileSnapshotController`